src/main/java/flight/reservation/plane/AirCraftModel.java

```diff
@@ -0,0 +1,11 @@
+package flight.reservation.plane;
+
+public enum AirCraftModel{
+    H1, //Helicopter
+    H2, //Helicopter
+    HypaHype, //PassengerDrone
+    A380, //Aircraft
+    A350, //Aircraft
+    Embraer_190, //Aircraft
+    Antonov_AN2 //Aircraft
+}
```

Comment:
Consider splitting this up in one enum per AirCraft category. A potential benefit is that you don't have to change this global list in case you add only, e.g., a helicopter